server/shared/auth/index.js

```diff
@@ -1,8 +1,8 @@
 'use strict';
 
-const { ExtractJwt, Strategy } = require('passport-jwt');
 const Audience = require('./audience');
-const config = require('../../../config/server/auth');
+const config = require('../../../config/server').auth;
+const { ExtractJwt, Strategy } = require('passport-jwt');
```

Comment:
I think multiple destructuring has higher priority import order and in this case it should be on top of everything. @vladimyr why did you move it?

Reply:
Didn't anticipate that we are following ESLint's import sorting convention :upside_down_face: Please move it up then.